package.json

```diff
@@ -32,6 +32,9 @@
     "@nestjs/config": "1.1.5",
     "@nestjs/core": "8.2.3",
     "@nestjs/platform-express": "8.2.3",
+    "class-transformer": "0.5.1",
```

Comment:
Deze zie ik nergens gebruikt worden, als je die in de toekomst nog gaat nodig hebben mag je die laten staan anders mag die nog weg.